storage/client/client.go

```diff
@@ -25,6 +26,8 @@ import (
 const (
 	blockCost = 1
 	txCost    = 1
+
+	maxTotalCount = 2000
```

Comment:
I'd guess we also expose this in documentation?

Reply:
Going to leave the actual 1000 out of the documentation for now; if we get lots of requests from users asking about it we can update it.